Sources/PagecallSDK/MiController.swift

```diff
@@ -213,4 +213,26 @@ class MiController: MediaController, SendTransportDelegate, ReceiveTransportDele
             return 0
         }
     }
+    
+    private var timer: Timer?
+    func startVolumeScheduler() {
+        timer?.invalidate()
+        timer = Timer.scheduledTimer(
+            timeInterval: 0.5,
+            target: self,
+            selector: #selector(emitVolume),
+            userInfo: nil,
+            repeats: true
+        )
+    }
+    
+    func stopVolumeScheduler() {
```

Comment:
[Minor]
명시적으로 호출해주지 않아도 deinit 에서 정리해주는 것이 좋을 것 같아요

```suggestion
    deinit {
      stopVolumeScheduler()
    }
    
    func stopVolumeScheduler() {
```